src/Method.php

```diff
@@ -110,7 +113,7 @@ public static function parse(\ReflectionMethod $method): Method
 
         $m = new self();
         $m->name = $method->getName();
-        $m->input = $method->getParameters()[1]->getClass()->getName();
+        $m->input = $method->getParameters()[1]->getType()->getName();
```

Comment:
This might be caught as a polymorphic call by code analyzers. Maybe use the same syntax as the one proposed above?